org.freedesktop.Platform.GL.nvidia.json.in

```diff
@@ -5,59 +5,11 @@
     "build-extension": true,
     "sdk": "org.freedesktop.Sdk",
     "runtime-version": "@@SDK_RUNTIME_VERSION@@",
-    "sdk-extensions": [],
     "separate-locales": false,
-    "cleanup": ["/include"],
-    "build-options" : {
-        "append-ld-library-path": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/lib",
-        "append-pkg-config-path": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/lib/pkgconfig",
-        "cflags": "-O2 -g -I/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/include",
-        "cxxflags": "-O2 -g",
-        "prefix": "/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@",
-        "env": {
-            "V": "1",
-            "LDFLAGS": "-L/usr/lib/GL/nvidia-@@NVIDIA_VERSION@@/lib"
-        }
-    },
     "finish-args": [
-        "--extra-data=@@EXTRA_DATA@@",
-        "--metadata=Extra Data=NoRuntime"
```

Comment:
Hmm, I removed this option because otherwise the `apply_extra` binary wouldn't run, as it links against libraries from the 24.08 runtime.

But now that I think about it, requiring the 24.08 runtime to be installed for the `apply_extra` binary to run doesn't make sense, so I'm afraid I'll still have to build libzstd and libarchive statically...

Marking as draft until that is done.